### PR TITLE
Profile page with graphql quering

### PR DESCRIPTION
**Pull request by GarrySan**

### CODE REVIEW

**Comment by prampey** [comment]
#82

File: swd/frontend/src/Routes/profile/Profile.js [hunk 1]
```diff
@@ -6,13 +6,22 @@ import s from "./Profile.css";
...
+            return <h4>loading..</h4>
+        }
+        if(error){
+            return <h4>Some error occured</h4>
+        } 
+        console.log(this.props.data);
         return (
             <Mobile>
                 <div className={s.container}>
-                    <h2>Profile</h2>
```
Comment (prampey) [inline swd/frontend/src/Routes/profile/Profile.js]: Leave the Profile header according to the Figma design.

File: swd/frontend/src/Routes/profile/Profile.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+            <Mobile>
+                <div className={s.container}>
+                    <h2>Profile</h2>
+                    <div className={s.profileImg}>
+                        <CardMedia>
+                            <img src={pic} style={{width: "100%", minWidth: "20%"}} alt="profile_pic"/>
+                        </CardMedia>
+                    </div>
+                    <div className={s.paperDiv}>
+                        <Paper zDepth={2} style={{borderRadius: 8, margin: 4, padding: 3}}>
```
Comment (prampey) [inline swd/frontend/src/Routes/profile/Profile.js]: The depth seems a bit too much visually. Try to make the shadows a bit more subtle. Use Paper or Card whichever gives the best result.

Comment (prampey) [reply swd/frontend/src/Routes/profile/Profile.js]: This seems to be the case for all the other cards too.

Comment (prampey) [reply swd/frontend/src/Routes/profile/Profile.js]: Also check if square looks better than rounded.

File: swd/frontend/src/Routes/profile/Profile.js (hunk 1 above)
Comment (aryanagarwal15) [reply swd/frontend/src/Routes/profile/Profile.js]: @GarrySan did you deliberately delete it? If so what was the reason?

**Comment by aryanagarwal15** [comment]
@prampey Please review now.

**Comment by prampey** [comment]
@aryanagarwal15 Please note that any conflicts (with master branch) should be resolved _after_ the intended functionality has been reviewed. By resolving a conflict in one of the commits, a breaking change has been introduced where the Search component cannot be found (as the current PR is yet to be integrated with the master branch).

**Comment by prampey** [comment]
I suggest the following workflow
- Review functionality of current issue at hand.
- Rebase with master to get linear commit history and solve any conflicts in the process
- Review and test the functionality again to make sure no breaking changes

**Comment by prampey** [comment]
Please note that 'rebase' rewrites public commit history so this should be done **only** when review has confirmed the PR for merge. Otherwise, we'd all have different commit signatures on our local branch during development.

**Pull request merged**